catalog/spatial/geocoding/spatial-geocoding-lucene/pom.xml

```diff
@@ -27,6 +27,16 @@
     <packaging>bundle</packaging>
```

Comment:
Should this be renamed from "spatial-geocoding-lucene"?

Reply:
Since we are adding support to query our own Solr core, it could be argued that the name need not change

Reply:
renamed to `api-impl` since the impl is no longer specific